### PR TITLE
Add San Diego Workshop folder

### PR DESCRIPTION
**Pull request by ameukam**

### CODE REVIEW

File: sandiego/OWNERS [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# See the OWNERS docs: https://go.k8s.io/owners
+
+reviewers:
+  - allisondy
```
Comment (guineveresaenger) [inline sandiego/OWNERS]:
```suggestion
  - alisondy
```

File: sandiego/OWNERS [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - ameukam
+  - chris-short
+  - guineveresaenger
+  - markyjackson-taulia
+  - mrbobbytables
+  - raelga
+approvers:
+  - allisondy
```
Comment (guineveresaenger) [inline sandiego/OWNERS]:
```suggestion
  - alisondy
```

**Comment by guineveresaenger** [comment]
I'll LGTM once Alison is spelled correctly 😛 thank you for this PR 🙏 
/approve

**Comment by mrbobbytables** [comment]
The [job adding them has completed](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/post-org-peribolos/1192196560339865600), but I think they need to accept the invite request first.

**Comment by mrbobbytables** [comment]
/assign guineveresaenger
for lgtm 👍

**Pull request merged**